src/style.css

```diff
@@ -1,8 +1,63 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
-    background-color: rgb(214, 214, 214);
-};
+  width: 95%;
+  margin: auto;
+}
+
+.container {
+  display: flex;
+  width: 80%;
+  justify-content: space-around;
+  margin: auto;
+}
+
+.left-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.left-contain,
+.right-contain {
+  flex-basis: 40%;
+}
+
+.right-contain h3 {
+  margin-bottom: 42px;
+}
+
+.right-contain {
+  text-align: end;
+}
+
+ul {
+  border: 2px solid black;
+  list-style: none;
+  padding-left: 0;
+  line-height: 28px;
+}
+
+ul li:nth-child(odd) {
+  background-color: rgb(180, 180, 180);
+}
+
+input {
+  width: 100%;
+  height: 40px;
+  margin-bottom: 8px;
+}
+
+.btn {
+  padding: 8px;
+  display: flex;
+  align-items: center;
+  height: 28px;
+}
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly adjust your styling to match the wireframe below:

*wireframe*
![matchhh](https://user-images.githubusercontent.com/59397159/173712957-824fbbac-111f-40f8-b968-4b61fe8ffd5e.png)

*your design*
![your_wireframe](https://user-images.githubusercontent.com/59397159/173713075-0a379db8-6522-483f-a704-65cc73ddd815.png)